scripts/us_cdc/environmental_health_toxicology/README.md

```diff
@@ -50,24 +50,24 @@ The dataset documentation is accessible on the [CDC National Evnvironment Public
 ### Artifacts
 
 #### Scripts
-[`parse_air_quality.py`](data/scripts/us_cdc/environmental_health_toxicology/)
+[`parse_air_quality.py`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/scripts/parse_air_quality.py)
 
-[`parse_precipitation_index.py`](data/scripts/us_cdc/environmental_health_toxicology/)
+[`parse_precipitation_index.py`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/scripts/parse_precipitation_index.py)
 
```

Comment:
Also add a subsection that notes and links to the files needed for the test scripts and the expected output.

Reply:
done